modules/dns/main.tf

```diff
@@ -21,3 +21,10 @@ resource "ibm_cis_dns_record" "dns_records" {
   data      = ((each.value.data != null) && (each.value.type == "SRV")) ? merge(each.value.data, { "name" : lookup(each.value, "name", "") }) : lookup(each.value, "data", null)
   proxied   = lookup(each.value, "proxied", false)
 }
+
+resource "ibm_cis_dns_records_import" "dns_record_import" {
+  count     = var.dns_records_file != null ? 1 : 0
+  cis_id    = var.cis_instance_id
+  domain_id = var.domain_id
+  file      = var.dns_records_file
```

Comment:
It is reading file from local. We discussed before that it does not work in case of schematics. Let me know if you tested this one.

Reply:
Resolved